research/python/activate_env.sh

```
@@ -0,0 +1,26 @@
+#!/bin/bash
+# Galactus Research Python Environment Activator
+
+echo "Activating Galactus Python virtual environment..."
+cd "$(dirname "$0")"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)